website/client/src/components/groups/questSidebarSection.vue

```diff
@@ -424,8 +424,9 @@ export default {
     },
     async questAccept (partyId) {
       const quest = await this.$store.dispatch('quests:sendAction', { groupId: partyId, action: 'quests/accept' });
+      quest.progress.up = this.user.party.quest.progress.up;
```

Comment:
@rogesson is this line necessary? what does it help with?

Reply:
Oh I see, the user's quest object is reset with the party object which lacks the damage info, but @rogesson I think this only works for boss quests not for collection ones for example here https://github.com/HabitRPG/habitica/blob/84869ff6078a37c5cb5d849212292dab3bf1b898/website/client/src/components/groups/questSidebarSection.vue#L116, maybe it'd be better to sync the entire user and not rely on the party quest info to replace this.user.party.quest

cc @Alys

Reply:
Hello @paglias

Yeah! The `progress.up` was not being displayed correctly.

After your comment, I removed the assignment and created a new validation based on the example you had sent me.

Could you check again? Thank you!